cgp_seq_input_val/seq_validator.py

```diff
@@ -311,3 +329,9 @@ def setup_progress(self):
         print("Progress is %d's of record pairs" % (self.progress_pairs), file=sys.stderr)
         bar.update(0)
         return bar
+
+
+def get_fq_format(file_h):
+    read = FastqRead(file_h, 0, None)
+    file_h.seek(0)  # reset file pointer to the begaining
+    return read.get_fq_format()
```

Comment:
Can you configure your editing tools to ensure you have a new line at the end of text files please?

Reply:
Can see the issue if you view the main diff.

Reply:
Set in my vscode now.